Dev4Agriculture.ISO11783.ISOXML.Test/StreamInputTests.cs

```diff
@@ -10,7 +10,23 @@ public class StreamInputTests
     [TestMethod]
     public void CanLoadValidZipStreamWithGrids()
     {
-        var filePath = "./testdata/LoadFromStream/Zipped_Task.zip";
+        var filePath = "./testdata/LoadFromStream/ZippedTask.zip";
+        ISOXML result = null;
+        using (var stream = File.OpenRead(filePath))
+        {
+            result = ISOXML.LoadFromArchive(stream);
+        }
+
+        Assert.IsNotNull(result);
+        Assert.IsNotNull(result.Data);
+        Assert.AreEqual(1, result.Grids.Count);
+        Assert.AreEqual(1, result.Grids["GRD00001"].Layers);
+    }
+
+    [TestMethod]
+    public void CanLoadValidZipStreamWithGridsFromSubFolder()
```

Comment:
This one crashes currently because the Lib does not respect SubFolders. Please fix :)